### PR TITLE
Added my blog entry 

### PR DESCRIPTION
**Pull request by waqassadiq**
Hi, I have added my blog details in blogs.html and also made a change in index.html regarding what i have learned in git.

Let me know if you require anything else from me.

Kind Regards,
Waqas Sadiq

### CODE REVIEW

File: blogs.html [hunk 1]
```diff
@@ -40,6 +40,19 @@
...
+                  <div class="timelineBlog-heading">
+                      <h4 class="timelineBlog-title">Waqas Sadiq (@Waqas) - Time Traveling in the Git World of Developers
+          <a href="https://www.linkedin.com/pulse/time-traveling-git-world-developers-waqas-sadiq/?published=t"><i class="glyphicon glyphicon-link"></i></a></h4>
+                  </div>
+                  <div class="timelineBlog-body">
+                      <p>In this post, I shared my beautiful learning experience with Udacity and how the knowledge gained has helped me improve my Git and Github skills relating it to a past experience. I briefly touched on Git Tagging, Branching and merging. Enjoy...</p>
+                      <p><small class="text-muted pull-right">Added On 22/01/2020</small></p>
+                  </div>
+              </div>
+          </li>
```
Comment (Titan-Programmer) [inline blogs.html:55]: Excellent here, well done.

File: index.html [hunk 2]
```diff
@@ -75,6 +75,7 @@ <h4>Git and GitHub</h4>
                                     <img src="images/GitHubGit.jpeg" class="img-responsive center-block"
                                         style="height: 150px;" alt="no-image.png">
                                     <ul style="padding-top: 20px;">
+                                        <li class="no-borders-list">Worked with pull requests</li>
```
Comment (Titan-Programmer) [inline index.html]: Thanks for the contribution but I don't believe the extra line is required. 

However, from what your trying to add it would be better to instead add Pull Requests in the line **Tagged, branched and merged** to something like **Tagged, branched, pull requests and merges**. I'm open to further suggestions.

Could you please make the required changes, commit and make another pull request?

Thanks.

**Review by Titan-Programmer — CHANGES_REQUESTED** [review]
Please see the comments attached. 

Many thanks.

File: blogs.html (hunk 1 above)
Comment (waqassadiq) [reply blogs.html:55]: thank you

File: index.html (hunk 2 above)
Comment (waqassadiq) [reply index.html]: I am making another pull request, in that, I have removed the extra line and added a comment in the tab section.

**Review by waqassadiq — COMMENTED** [review]
I am making this pull request to fix the extra line and also proper comment on git tab on index page. thank you.

**Review by Titan-Programmer — APPROVED** [review]
Excellent great work!

Please feel free to add to the additional tabs and spread the word so other students can practice their Git skills.
